app/javascript/packages/verify-flow/index.ts

```diff
@@ -6,9 +6,6 @@ export { default as Cancel } from './cancel';
 export { default as VerifyFlow } from './verify-flow';
 export { default as VerifyFlowStepIndicator, VerifyFlowPath } from './verify-flow-step-indicator';
 
-export { default as personalKeyStep } from './steps/personal-key';
-export { default as personalKeyConfirmStep } from './steps/personal-key-confirm';
```

Comment:
this just removes the imports, but doesn't remove the steps themselves. are those still being used?

Reply:
They're still being used, but the usage is internal to the `verify-flow` package. These were exported here so that they could be referenced directly in the `verify-personal-key` pack being removed here.